### PR TITLE
Added new responsive log in form (ready to use)

### PR DESCRIPTION
**Pull request by balayokesh**
I created a new folder named forms, also included one responsive login form in it.  I hope this form template would be useful for others.

### CODE REVIEW

**Comment by krutarththakkar** [comment]
Good one!
Thanks, and to point out scopes for improvement,
you can really work on the design and looks
and help the community by contributing.
But anyways this was a good piece work, and I acknowledge the efforts.

**Comment by balayokesh** [comment]
Thanks for giving some suggestions.  Now I included a contact form please take a look at it

File: Code Snippets/Designs/Form designs/contactform.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+
```
Comment (krutarththakkar) [inline Code Snippets/Designs/Form designs/contactform.html:1]: You can comment if you have any tutorials link here

**Comment by krutarththakkar** [comment]
Sure, Thanks again!
And do ✨ the repository if you liked it!

**Pull request closed without merging**